ansible/roles/test/files/ptftests/vxlan-decap.py

```diff
@@ -188,10 +238,9 @@ def warmup(self):
         ret = 0
         try:
             for test in self.tests:
-                if self.vxlan_enabled:
-                    self.Vxlan(test, True)
                 self.RegularLAGtoVLAN(test, True)
-                self.RegularVLANtoLAG(test, True)
+                #wait sometime for DUT to build FDB and ARP table
+                time.sleep(60)
```

Comment:
I set the delay to 60 seconds because there is a noticeable delay between TCP packets are sent in RegularLAGtoVLAN and ARP table is built. I'm sorry I can't give a precise figure about the delay, so I set it to 60 that is long enough. But the total time for warmup is much shorter because we remove some operation from warmup. So I think it's acceptable.

Reply:
Usually, when we don't control something in the test and set a timer for "big" value, that could make the test fail sometimes. Is it possible to modify the test to poll when it is a time to go?

Reply:
Thanks pavel. It's a good idea. I'll think about it and post an update soon.

Reply:
@pavel-shirshov 
A polling check is added to ptf script to check whether FDB and ARP table are ready. It is more stable and fast now. Please help to review. Thanks!